packages/data-stores/src/index.ts

```diff
@@ -30,7 +30,6 @@ export * from './templates';
 export * from './onboard/types';
 export * from './domain-suggestions/types';
 export * from './plans/types';
-export * from './subscriber/types';
```

Comment:
Nice, those are already exported as `Subscriber.` types and consumed only that way 👍